inst/examples/index.Rmd

```diff
@@ -612,6 +612,23 @@ Table \@ref(tab:test-table):
 knitr::kable(head(iris[, -5]), caption = 'An example table.')
 ```
 
+## HTMLWidgets
+
+HTMLWidgets keep their interactivity: you can insert a widget, interactively modify it and then print the document.^[This interactivity is not allowed in RStudio because of security restrictions.] As any other other figures, you can modify their dimensions and styles using the chunk options `out.width`, `out.height` and `out.extra`.  
```

Comment:
```suggestion
HTMLWidgets keep their interactivity: you can insert a widget, interactively modify it and then print the document.^[This interactivity is not allowed in RStudio because of security restrictions.] As any other figures, you can modify their dimensions and styles using the chunk options `out.width`, `out.height` and `out.extra`.  
```

Reply:
Thanks!